tsconfig.json

```diff
@@ -40,5 +40,8 @@
   "exclude": [
     "node_modules",
     "dist"
-  ]
+  ],
+  "vueCompilerOptions": {
+    "target": 2.7
+  }
```

Comment:
This is to solve the following "known issue":

![image](https://user-images.githubusercontent.com/10002889/194602803-9b4af42c-fffd-4265-afa7-deeb96bc0a55.png)